doc/changes/changes_1.0.0.md

```diff
@@ -103,3 +104,21 @@ Relocked dependencies
 ## Bugfix
 
 * #183: Replaced terminate_export & terminate_import for ExaHttpThread with terminate
+
+## Dependency Updates
+
+### `main`
+* Updated dependency `orjson:3.10.6` to `3.10.18`
+* Updated dependency `packaging:24.1` to `25.0`
+* Updated dependency `pandas:2.2.2` to `2.3.0`
```

Comment:
do we want to prepend optional here? atm PTB doesn't distinguish optional vs required.